### ISSUE TITLE
Weird text problem

### ISSUE THREAD

**Issue by raincoated** [new-xkit/XKit#2073]
So whenever I open a new text post, the little blinking line that follows every letter you type in will disappear after a second, making it as though i've clicked off the text post. So when I go to type, nothing types. I have to click the box first. If im quick enough, i can get about 4 letters in before it decides to stop working. Uninstalling xkit fixes this problem. Ive uninstalled and reinstalled so many times.

Also, whenever I upload an image in a text post and type "hello" for example itll come out as "elloh".  

I have the latest version of xkit and I'm using chrome. Everything is up to date. Please help :(

**Comment by marcustyphoon** [comment]
Hm, I have that second problem regardless of whether I have xkit installed or not, if I type fast enough after uploading an image. The first problem, I haven't been able to reproduce.

Are you reinstalling XKit and importing your settings from XCloud, or do you experience this with just the default extensions? Do you experience it with XKit installed but every single extension disabled? See if you can track down which extension causes it.

**Comment by raincoated** [comment]
@marcustyphoon 
actually, yeah, the second problem happens even with xkit uninstalled and its only when answering Asks. Guess that's just a tumblr problem.

And yeah when I reinstalled xkit i didnt import anything, it was all just default stuff. I tried to track down which extension but I couldn't find anything.

**Comment by marcustyphoon** [comment]
Hm. I assume you're not using the beta post editor, then - if I turn it off and install editable reblogs, I do experience the post compose box becoming unselected, but only once right after I click to make a new post. Haven't gotten it to occur more than once yet. The editable reblogs code is all 5-6 years old and fairly, uh, interesting, so while I don't see an obvious cause, I don't exactly... not.

Do you have any other browser extensions? Does this occur with nothing but XKit enabled?

**Comment by raincoated** [comment]
@marcustyphoon 
Yeah it only deselects the one time as soon as i open a new text post. And I have the beta post editor turned off. I turned off the other extensions that i have like Better Scrollbar and Honey and it's pretty much just editable reblogs that is causing this. If I turn xkit off, it works. I mean, I guess I can just keep turning it off unless I need to reblog something but that's so inconvenient.

**Comment by marcustyphoon** [comment]
Wait, I don't understand... if it only happens once, and you can just click in the box to continue your post, why would you need to disable XKit?

**Comment by raincoated** [comment]
Anyway, thanks for the help. I at least ended up finding out what was causing the issue and I can work around it. Idk how i didnt find the issue the last time i tried to figure it out, i guess i didn't go through every installed extension and just disabled xkit. Oh well, whatever, I got it now lol
